From 6c0ab00375412507cf5cd5b7de58e99e2f5be516 Mon Sep 17 00:00:00 2001
From: port-worker <port-worker@local>
Date: Tue, 11 Aug 2026 08:42:52 +0000
Subject: [PATCH 03/27] init: Do not start console service when debuggable

Google added a check for this in R, when it's running it will show a
notification about that performance is impacted.

Signed-off-by: Isaac Chen <tingyi364@gmail.com>
Change-Id: I34cfd6b42d3b9aee4b3e63181480cfb8b1255f29
[forward-ported to A16: block moved to line 1367, trailing context is now Multi-Gen LRU]
---
 rootdir/init.rc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 92dcdd013..f9ff5427a 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -1364,9 +1364,6 @@ on property:ro.debuggable=1
     # Give reads to anyone for the accessibility trace folder on debug builds.
     chmod 0775 /data/misc/a11ytrace
 
-on init && property:ro.debuggable=1
-    start console
-
 # Multi-Gen LRU Experiment
 on property:persist.device_config.mglru_native.lru_gen_config=none
   write /sys/kernel/mm/lru_gen/enabled 0
-- 
2.43.0

